src/plugins/irc-events/link.js

```diff
@@ -52,35 +42,35 @@ function parse(msg, url, res, client) {
 	switch (res.type) {
 	case "text/html":
 		var $ = cheerio.load(res.text);
-		toggle.type = "link";
-		toggle.head =
+		preview.type = "link";
+		preview.head =
 			$("meta[property=\"og:title\"]").attr("content")
```

Comment:
Fair enough, we'll see at usage if that's something that would make sense or not